Source/Blazorise/Components/Accordion/Accordion.razor.cs

```diff
@@ -24,6 +27,35 @@ protected override void BuildClasses( ClassBuilder builder )
         base.BuildClasses( builder );
     }
 
+    internal void NotifyCollapseInitialized( Collapse collapse )
+    {
+        collapses ??= new();
+
+        collapses.Add( collapse );
+    }
+
+    internal void NotifyCollapseRemoved( Collapse collapse )
+    {
+        if ( collapses is not null && collapses.Contains( collapse ) )
+            collapses.Remove( collapse );
+    }
+
+    internal bool IsFirstInAccordion( Collapse collapse )
+    {
+        if ( collapses is not null && collapses.IndexOf( collapse ) == 0 )
+            return true;
+
+        return false;
```

Comment:
basically `return ( collapses is not null && collapses.IndexOf( collapse ) == 0 )` if we want to be picky. Same for the one below this.